app/src/main/java/com/bootcamp/kotlin/welcome/SplashActivity.kt

```diff
@@ -35,33 +32,42 @@ class SplashActivity : AppCompatActivity() {
 
     private fun sleepScreen() {
         /****** Create Thread that will sleep for 2 seconds */
-        Handler().postDelayed({
+        handler.postDelayed({
             checkIfUserExists()?.let {
                 if (it.isEmpty()) {
                     showMessage("No hay usuarios registrados")
-                    loadFragment(RegisterFragment(), R.id.registerFragment)
+                    R.id.splashContainer.attachFragment(
+                        RegisterFragment(),
+                        getString(R.string.tag_register_fragment)
```

Comment:
en vez de crear strings yo usaría el nombre del fragment cuando estes haciendo el replace:

`.replace(this, view, view.javaClass.simpleName)`

Reply:
es un tag mas no el nombre del fragment entonces si deberia ir alla